packages/datatrak-web/src/views/LandingPage/ActivityFeedSection/ActivityFeedSurveyItem.tsx

```diff
@@ -0,0 +1,75 @@
+/*
+ * Tupaia
+ *  Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+
+import React from 'react';
+import styled from 'styled-components';
+import { format } from 'date-fns';
```

Comment:
Good idea adding date-fns. We are in a bit of a transition time as we still have moment in ui-components but I think that it makes sense to draw a line and use date-fns in datatrak-web. Unfortunately we have already installed moment and used it in a couple of places in the app. Would you mind doing a follow up PR or creating a ticket to switch out the other uses of moment in datatrak-web to date-fns. Better sooner rather than later.

Reply:
I actually ended up using the built in JS date in the second part of this - it was better for locale date strings I found

Reply:
ok that's a good point. I think that we should make a date display component but I just remembered that Georgia is going to make a ticket for it so we can figure that out later :)